### ISSUE TITLE
no conv bias with BatchNorm

### ISSUE THREAD

**Issue by troyliu0105** [tristandb/EfficientDet-PyTorch#6]
in this line https://github.com/tristandb/EfficientDet-PyTorch/blob/b86f3661c9167ed9394bdfd430ea4673ad5177c7/bifpn.py#L37
should we disable conv's bias when the bn is used with it?
